rudder-core/src/test/resources/configuration-repository/expected-share/node-cfe-with-two-directives/rules/cfengine-community/rudder-directives.cf

```diff
@@ -35,6 +35,7 @@ body file control {
 bundle agent rudder_directives {
 
   methods:
+      "pre-run-hook"                                  usebundle => do_run_hook("package","","{"parameters":{"package":"vim"},"reports":[{"id":"208716db-2675-43b9-ab57-bfbab84346aa@@16d86a56-93ef-49aa-86b7-0d10102e4ea9@@0","mode":"enforce"}]}");
```

Comment:
the quoting here looks dodgy
"{"parameters":{"package":"vim"},"reports":[{"id":"208716db-2675-43b9-ab57-bfbab84346aa@@16d86a56-93ef-49aa-86b7-0d10102e4ea9@@0","mode":"enforce"}]}"

Reply:
what is the correct quoting?

Reply:
i guess replacing the inner double quotes by simple quotes (or the other way round)
so 
"{'parameters':{'package':'vim'},'reports':[{'id':'208716db-2675-43b9-ab57-bfbab84346aa@@16d86a56-93ef-49aa-86b7-0d10102e4ea9@@0','mode':'enforce'}]}"
or
'{"parameters":{"package":"vim"},"reports":[{"id":"208716db-2675-43b9-ab57-bfbab84346aa@@16d86a56-93ef-49aa-86b7-0d10102e4ea9@@0","mode":"enforce"}]}'